.claude/hooks/protect-claude-dir.sh

```
@@ -0,0 +1,47 @@
+#!/bin/bash
+# PreToolUse hook to explain why .claude directory access is forbidden
+#
+# This hook provides a helpful error message when Claude attempts to
+# read, edit, or write files in the .claude directory.
+
+input=$(cat)
+
+cd "$CLAUDE_PROJECT_DIR"
+
+# Extract the tool name and file path from hook input
+tool_name=$(echo "$input" | jq -r '.tool_name // empty')
```

Shellcheck (0.11.0)

[warning] tool_name appears unused. Verify use (or export if used externally).

(SC2034)